src/snowflake/cli/_plugins/helpers/commands.py

```diff
@@ -35,10 +36,24 @@ def v1_to_v2(
     accept_templates: bool = typer.Option(
         False, "-t", "--accept-templates", help="Allows the migration of templates."
     ),
+    migrate_local_yml: bool = typer.Option(
+        False,
+        "-l",
+        "--migrate-local-overrides",
```

Comment:
we might have to wordsmith this to explain that we'll include it in the conversion but not produce a separate file / a separate set of entities for it.

Reply:
updated